packages/site/src/locales/bn.json

```diff
@@ -188,20 +190,20 @@
     "edit": "সম্পাদনা করুন",
     "invalid": "অবৈধ",
     "next": "পরবর্তী",
-    "export": "এক্সপোর্ট",
-    "images": "ছবিগুলো",
+    "export": "রফতানি",
+    "images": "চিত্র",
     "reset": "রিসেট",
-    "invalid_url": "এটি একটি বৈধ URL নয়",
-    "downloading": "ডাউনলোড হচ্ছে",
-    "upload": "আপলোড করুন"
+    "invalid_url": "এটি কোনও বৈধ ইউআরএল নয়",
+    "downloading": "ডাউনলোড করা",
+    "upload": "আপলোড"
   },
   "settings": {
     "community_permission": "বক্তৃতা সম্প্রদায় কী এই ভাষাটিকে অনলাইনে থাকতে দেয়?",
     "edit_dict_name": "অভিধানের নাম সম্পাদনা করুন",
     "dict_name": "অভিধানের নাম",
     "public_private_meaning": "সর্বজনীন মানে যে কেউ আপনার অভিধান দেখতে পারে যার জন্য সম্প্রদায়ের সম্মতি প্রয়োজন। ব্যক্তিগত অভিধানগুলি কেবল আপনার এবং আপনার সহযোগীদের কাছে দৃশ্যমান।",
     "optional_data_fields": "ঐচ্ছিক তথ্য ক্ষেত্র",
-    "unable-deleting-msg": "দ্রষ্টব্য: আপনি পরে এই glossing ভাষা মুছে ফেলতে পারবেন না।"
+    "unable-deleting-msg": "দ্রষ্টব্য: আপনি পরে এই চকচকে ভাষাগুলি মুছতে পারবেন না।"
```

Comment:
@Danble I missed seeing this key come in originally - can you please update to be "unable_to_delete"? Thank you, this will keep the use of underscores and not hyphens to be consistent. As well, "msg" is not needed as these are all messages.